### PR TITLE
Switch desktops to KDE

### PR DESCRIPTION
**Pull request by ethanwu10**
# Open KDE Facility

**Why?**
We got 4K monitors and scaling on XFCE, GNOME, and Cinnamon all don't support fractional scaling properly. (Also, I like KDE :D)

TODOs:
- [x] set scaling back to 1 for non-HiDPI monitors
- [x] scale the display manager
- [ ] fix GTK3 theme having minimize/maximize/close buttons in CSDs being invisible
- [ ] switch remaining GTK apps to KDE ones (e.g. gedit)
- [x] session tracking broke (fixed)
- [x] kill inactive-but-locked sessions *somehow*

### CODE REVIEW

**Comment by Kalissaac** [comment]
also the session tracking doesn't work on KDE, it opens a tty and multiple pts which breaks the script when it runs `who`

**Comment by cg505** [comment]
holy

File: hieradata/type/desktop.yaml [hunk 1]
```diff
@@ -2,7 +2,9 @@ classes:
     - ocf_desktop
 
 ocf::auth::glogin: [[ocf, LOCAL]]
-ocf::auth::ulogin: [[lightdm, LOCAL]]
+ocf::auth::ulogin: [[lightdm, LOCAL], [sddm, LOCAL]]
```
Comment (nikhiljha) [inline hieradata/type/desktop.yaml:5]: Do we still need SDDM?

File: modules/ocf_desktop/files/xsession/Xsession [hunk 2]
```diff
@@ -47,9 +49,6 @@ sleep 8 && /usr/local/bin/fix-audio &
 # Auto logout inactive users
 /usr/bin/xautolock -locker /usr/local/bin/auto-lock -time 7 -noclose &
```
Comment (nikhiljha) [inline modules/ocf_desktop/files/xsession/Xsession:50]: Is this doing anything anymore?

**Review by nikhiljha — APPROVED** [review]
lgtm, may want to look into some things though

**Comment by cg505** [comment]
woah

**Comment by nikhiljha** [comment]
@cg505 this has been running in prod for months already lol

**Comment by cg505** [comment]
imagine going to the lab

**Pull request merged**